xml/System/DateTime.xml

```diff
@@ -6825,9 +6825,11 @@ The following example demonstrates the <xref:System.DateTime.Parse%28System.Stri
 ## Remarks  
  A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond, or 10 million ticks in a second.  
   
- The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 (0:00:00 UTC on January 1, 0001, in the Gregorian calendar), which represents <xref:System.DateTime.MinValue?displayProperty=nameWithType>. It does not include the number of ticks that are attributable to leap seconds.  
+ The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 (0:00:00 on January 1, 0001, in the Gregorian calendar, in the current time zone), which represents <xref:System.DateTime.MinValue?displayProperty=nameWithType>.It does not include the number of ticks that are attributable to leap seconds.  
```

Comment:
>in the current time zone) [](start = 193, length = 25)

I don't think this is true, DateTime.Kind can specify Utc, Local, or Unspecified. so this depend on Kind property inside the DateTiem object